### PR TITLE
Meal-Sharing homework react1-week3

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**

### CODE REVIEW

File: src/client/components/MealList.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                </p>
+                <p>Description: {aMeal.description}</p>
+                <p>Price: {aMeal.price}</p>
+              </li>
+            );
+          })}
+        </ul>
+      ) : (
+        <p>Loading... </p>
+      )}
```
Comment (Divyajg) [inline src/client/components/MealList.jsx:36]: You can use `IsLoading` to display `Loading....`.

**Review by Divyajg — APPROVED** [review]
Good work.

File: src/client/components/MealList.jsx (hunk 1 above)
Comment (RodrigoGuinsbergPinto) [reply src/client/components/MealList.jsx:36]: Hi Divyaj,
thank you so much for reviewing my homework.
I made some implementation following your suggestions.

**Review by Divyajg — APPROVED** [review]
👍

File: src/client/components/MealList.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+
+const MealList = () => {
+  const [meal, setMeal] = useState();
```
Comment (shpomp) [inline src/client/components/MealList.jsx]: A more accurate naming would be essential.
It is an array of meals, so meals and setMeals 🙏🏽

File: src/client/components/MealList.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      setMeal(result);
+      setIsLoading(false);
+    })();
+  }, []);
+
+  return (
+    <div>
+      {meal ? (
+        <ul>
+          {meal.map((aMeal) => {
```
Comment (shpomp) [inline src/client/components/MealList.jsx]:
```suggestion
          {meals.map((meal) => {
```

**Review by shpomp — APPROVED** [review]
⭐

File: src/client/components/MealList.jsx (hunk 3 above)
Comment (RodrigoGuinsbergPinto) [reply src/client/components/MealList.jsx]: Hi Karo,
thanks for the comment. I implemented the code.

**Pull request merged**